src/components/MTableHeader/index.js

```diff
@@ -140,11 +140,18 @@ export function MTableHeader({ onColumnResized, ...props }) {
             <TableSortLabel
               IconComponent={props.icons.SortArrow}
               active={props.orderBy === columnDef.tableData.id}
-              direction={props.orderDirection || 'asc'}
+              direction={
```

Comment:
Can we combine these tw osections of determining the direction into one variable above the render so we dont have the same logic at several location?

Reply:
Seems complicated. Indeed, they all determine the direction but not at the same moment: what we calculate for "direction" (line 143) is the calculated orderDirection, and what we calculate for orderDirection in the onClick function (line 150) is the future orderDirection. First one is consequence of second one. I did not find a way to calculte them the same way.
I am opened to your suggestions for that. But after spending time on that, I have not found solution.

Reply:
True, maybe it would help to either create a new function that generates the next or current sorting direction. Also some comments would be needed to explain each step and how the work together so that we can maintain this in the future without figuring out under which condition which string is returned.